data/pages.yaml

```diff
@@ -53,6 +49,56 @@ novice:
   learn: Understand the fundamentals of Ansible automation
   install: Install the Ansible package
   start: Write your first playbook in a few easy steps
+# Resources for user journeys
+user:
+  title: User resources
+  intro: Automate the management of remote systems and control their desired state.
+  milestone:
+    automate: Create automation
+    build_inventory: Build inventories
+    organize: Organize automation projects
+    tooling: Use Ansible tooling
+    reuse: Re-use automation content
```

Comment:
Find automation content?  to distinguish this from creating roles (reusable content)